docker-compose-rule-junit4/src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -38,13 +38,15 @@
 import org.immutables.value.Value;
 import org.joda.time.Duration;
 import org.joda.time.ReadableDuration;
-import org.junit.rules.ExternalResource;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 @Value.Immutable
 @CustomImmutablesStyle
-public abstract class DockerComposeRule extends ExternalResource {
```

Comment:
Super technically this is a break, but I'm planning on making a (small) actual break tomorrow, so maybe once we merge this don't release and I'll group them together into a major version bump release at the same time?

Reply:
Sounds good to me

Reply:
The methods in `ExternalResource` are non-final, so we could avoid a break here by keeping the extends and overriding the methods

Reply:
Done